### DISCUSSION TITLE
Allow Static inside method

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/csharplang#6254] [Language Ideas] [answered]
Declaring a variable inside a method as static meaning it's gonna have the value it is initially asigned to inside the method to appear as static for all subsequent calls while only visible inside the method

**Comment by gfoidl** [comment]
See https://github.com/dotnet/csharplang/discussions/5057, https://github.com/dotnet/csharplang/issues/188

**Comment by svick** [comment, marked as answer]
This is a duplicate of https://github.com/dotnet/csharplang/discussions/832.
